tests/test_dag.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from airflow import DAG
+
+from FastETL.operators.copy_db_to_db_operator import CopyDbToDbOperator
+
+default_args = {
+    'owner': 'nitai',
+    'start_date': datetime(2020, 8, 20),
+}
+with DAG(
+        "test_dag",
+        default_args=default_args,
```

Comment:
```suggestion
        default_args=default_args,
        catchup=False,
```
Não seria melhor sem [catchup](https://airflow.apache.org/docs/apache-airflow/stable/dag-run.html#catchup) 🥪 ?